### DISCUSSION TITLE
Can I remove the aa-verify TXT record from my domain after successfully verifying a custom domain?

### DISCUSSION THREAD

**Discussion by jakeswenson** [anonaddy/anonaddy#558] [Q&A] [answered]
I've setup a custom domain, verified, it and it's been working great. I'm wondering if it's safe to remove the verification TXT record from my domain now or if anonaddy periodically continues to check for this record. I couldn't find anything in the FAQ on this, so that might be something worth adding..

**Comment by willbrowningme** [comment, marked as answer]
Yes you can remove the TXT verification record `aa-verify...`. 

I've just added a note about this when adding a new domain "Once you've added the domain you can safely remove this TXT record.".
